addons/sys_core/initSettings.sqf

```diff
@@ -11,17 +11,27 @@
 [
     QGVAR(premixGlobalVolume),
     "SLIDER",
-    "Pre-Mix Global Volume",
+    localize LSTRING(premixGlobalVolume_displayName),
     "ACRE2",
     [0, 3, 1, 2],
     false,
     {["premixGlobalVolume", _this] call FUNC(setPluginSetting)}
 ] call CBA_Settings_fnc_init;
 
 [
+    "ACRE_SPECTATOR_VOLUME",
+    "SLIDER",
+    localize LSTRING(ACRE_SPECTATOR_VOLUME_displayName),
+    "ACRE2",
+    [0, 1, 1, 2],
+    false,
+    {["premixGlobalVolume", _this] call FUNC(setPluginSetting)}
```

Comment:
This will overwrite the actual premix volume and will clash with the  above settings when sent to TS.